### PR TITLE
[WIP] Improve content in peripherals section

### PR DESCRIPTION
**Pull request by hamishwillee**
Add structure and more content to peripherals section. Reduce content in concepts section as required - for example sensors doc is "sensors overview" not "sensors selection".

### CODE REVIEW

**Comment by hamishwillee** [comment]
@bkueng I need to push this structure through this evening. This won't break anything, but I'd still appreciate your comments.

**Comment by hamishwillee** [comment]
Yes, but there is more coming, and doubtless some questions :-)

File: en/sensor/airspeed.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Airspeed sensors are *highly recommended* for fixed-wing and VTOL frames.
+They are so important because the autopilot does not have other means to detect stall.
+For fixed-wing flight it is the airspeed that guarantees lift not ground speed!
+
+![Digital airspeed sensor](../../images/digital_airspeed_sensor.jpg)
+
+
+## Hardware Options
+
+Recommended digital airspeed sensors include:
```
Comment (hamishwillee) [inline en/sensor/airspeed.md]: Are there any more we should mention? I added links that made sense based on the driver names and documentation.

**Comment by hamishwillee** [comment]
Merging now. Will iterate further over coming days.

File: en/sensor/airspeed.md (hunk 1 above)
Comment (bkueng) [reply en/sensor/airspeed.md]: This looks good. The MS4525DO is also available here: https://drotek.com/shop/en/airspeed/793-digital-differential-airspeed-sensor-kit-.html?search_query=airspeed&results=6.

**Pull request merged**